refactor: fix mypy issues in checkers/e*.py

Fix mypy errors for the following:

```
cve_bin_tool/checkers/exiv2.py:16: error: Need type annotation for "CONTAINS_PATTERNS" (hint: "CONTAINS_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/exiv2.py:17: error: Need type annotation for "FILENAME_PATTERNS" (hint: "FILENAME_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/exim.py:16: error: Need type annotation for "CONTAINS_PATTERNS" (hint: "CONTAINS_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/exim.py:17: error: Need type annotation for "FILENAME_PATTERNS" (hint: "FILENAME_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/elfutils.py:15: error: Need type annotation for "CONTAINS_PATTERNS" (hint: "CONTAINS_PATTERNS: List[<type>] = ...")
cve_bin_tool/checkers/elfutils.py:16: error: Need type annotation for "FILENAME_PATTERNS" (hint: "FILENAME_PATTERNS: List[<type>] = ...")
```

Helpful resources:

- [Python typing documentation](https://docs.python.org/3/library/typing.html)
- [mypy cheat sheet](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html)
- If you can't find the answer in the docs, googling the error messages above will often help!
- [cve-bin-tool's contributor docs](https://github.com/intel/cve-bin-tool/blob/main/CONTRIBUTING.md)
- If you've contributed to open source but not this project, you might just want our [checklist for a great pull request](https://github.com/intel/cve-bin-tool/blob/main/CONTRIBUTING.md#checklist-for-a-great-pull-request)

Tiny tips for new contributors:

- cve-bin-tool uses https://www.conventionalcommits.org/ style for commit messages, and we have a test that checks the title of your PR. A good potential title for this one is in the title of this issue.
- You can make an issue auto close by including a comment "fixes #ISSUENUMBER" in your PR comments where ISSUENUMBER is the actual number of the issue.

--- a/cve_bin_tool/checkers/e2fsprogs.py
+++ b/cve_bin_tool/checkers/e2fsprogs.py
@@ -9,12 +9,14 @@ https://www.cvedetails.com/product/12670/Ext2-Filesystems-Utilities-E2fsprogs.ht
 https://www.cvedetails.com/product/31107/E2fsprogs-Project-E2fsprogs.html?vendor_id=15251
 
 """
+
+from typing import List
 from cve_bin_tool.checkers import Checker
 
 
 class E2FsprogsChecker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
+    FILENAME_PATTERNS: List[str] = []
     VERSION_PATTERNS = [
         r"e2fsprogs\r?\n([0-9]+\.[0-9]+\.[0-9]+)",
         r"([0-9]+\.[0-9]+\.[0-9]+)\r?\nError: ext2fs",

--- a/cve_bin_tool/checkers/e2fsprogs.py
+++ b/cve_bin_tool/checkers/e2fsprogs.py
@@ -9,8 +9,8 @@ https://www.cvedetails.com/product/12670/Ext2-Filesystems-Utilities-E2fsprogs.ht
 https://www.cvedetails.com/product/31107/E2fsprogs-Project-E2fsprogs.html?vendor_id=15251
 
 """
-
 from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 

--- a/cve_bin_tool/checkers/elfutils.py
+++ b/cve_bin_tool/checkers/elfutils.py
@@ -8,12 +8,14 @@ CVE checker for elfutils
 https://www.cvedetails.com/product/27413/Elfutils-Project-Elfutils.html?vendor_id=13228
 
 """
+
+from typing import List
 from cve_bin_tool.checkers import Checker
 
 
 class ElfutilsChecker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
+    FILENAME_PATTERNS: List[str] = []
     VERSION_PATTERNS = [
         r"([0-9]+\.[0-9]+)\r?\nelfutils",
         r"elfutils\r?\n([0-9]+\.[0-9]+)",

--- a/cve_bin_tool/checkers/elfutils.py
+++ b/cve_bin_tool/checkers/elfutils.py
@@ -8,8 +8,8 @@ CVE checker for elfutils
 https://www.cvedetails.com/product/27413/Elfutils-Project-Elfutils.html?vendor_id=13228
 
 """
-
 from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 

--- a/cve_bin_tool/checkers/exim.py
+++ b/cve_bin_tool/checkers/exim.py
@@ -9,12 +9,14 @@ https://www.cvedetails.com/product/153/University-Of-Cambridge-Exim.html?vendor_
 https://www.cvedetails.com/product/19563/Exim-Exim.html?vendor_id=10919
 
 """
+
+from typing import List
 from cve_bin_tool.checkers import Checker
 
 
 class EximChecker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
+    FILENAME_PATTERNS: List[str] = []
     VERSION_PATTERNS = [
         r"exim/([0-9]+\.[0-9]+)",
         r"([0-9]+\.[0-9]+)\r?\n<<eximversion>>",

--- a/cve_bin_tool/checkers/exim.py
+++ b/cve_bin_tool/checkers/exim.py
@@ -9,8 +9,8 @@ https://www.cvedetails.com/product/153/University-Of-Cambridge-Exim.html?vendor_
 https://www.cvedetails.com/product/19563/Exim-Exim.html?vendor_id=10919
 
 """
-
 from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 

--- a/cve_bin_tool/checkers/exiv2.py
+++ b/cve_bin_tool/checkers/exiv2.py
@@ -9,11 +9,13 @@ https://www.cvedetails.com/product/6748/Andreas-Huggel-Exiv2.html?vendor_id=3849
 https://www.cvedetails.com/product/12768/Exiv2-Exiv2.html?vendor_id=7561
 
 """
+
+from typing import List
 from cve_bin_tool.checkers import Checker
 
 
 class Exiv2Checker(Checker):
-    CONTAINS_PATTERNS = []
-    FILENAME_PATTERNS = []
+    CONTAINS_PATTERNS: List[str] = []
+    FILENAME_PATTERNS: List[str] = []
     VERSION_PATTERNS = [r"exiv2 ([0-9]+\.[0-9]+\.?[0-9]*)"]
     VENDOR_PRODUCT = [("andreas_huggel", "exiv2"), ("exiv2", "exiv2")]

--- a/cve_bin_tool/checkers/exiv2.py
+++ b/cve_bin_tool/checkers/exiv2.py
@@ -9,8 +9,8 @@ https://www.cvedetails.com/product/6748/Andreas-Huggel-Exiv2.html?vendor_id=3849
 https://www.cvedetails.com/product/12768/Exiv2-Exiv2.html?vendor_id=7561
 
 """
-
 from typing import List
+
 from cve_bin_tool.checkers import Checker
 
 

